pkg/sbom/sbom.go

```diff
@@ -47,15 +65,17 @@ func SBOMWorkflow(
 		sbomDocs = append(sbomDocs, newData(depGraph, service.MimeTypeCycloneDXJSON, sbomBytes))
 	}
 
+	logger.Printf("Successfully generated SBOM documents: count %d\n", len(sbomDocs))
+
 	return sbomDocs, nil
 }
 
 func Init(e workflow.Engine) error {
 	flagset := pflag.NewFlagSet("snyk-cli-extension-sbom", pflag.ExitOnError)
-	// TODO: add proper user documentation.
-	flagset.String("file", "", "usage docs for file option")
-	// TODO: add proper user documentation.
-	flagset.StringP("format", "f", "text", "usage docs for format option")
+
+	flagset.Bool(flagExperimental, false, "Explicitly enable sbom command.")
+	flagset.String(flagFile, "", "Specify a package file.")
```

Comment:
I wonder if this is clear enough for users. What is a "package file" in the go ecosystem, for example? Maybe manifest file would be clearer? Or maybe "Specify a manifest file (e.g. package.json, go.mod, requirements.txt, etc.)? BTW, `snyk test --help` is not very consistent on this point. Sometimes it refers to manifest files and in other places it refers to package files.

Reply:
tbh, this is copy/paste from `snyk test --help` since rob trickey suggested this. i agree the naming is problematic. i'd like for us to revisit this wording with the help of product or technical writers.

Reply:
@mcombuechen I would suggest just changing the experimental flag warning to `Explicitly enable sbom command with the --experimental flag` - then the messaging is clear on what the user needs to do